web/apps/labelstudio/src/pages/CreateProject/Import/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { API } from "apps/labelstudio/src/providers/ApiProvider";
```

Comment:
If this function is meant to be a hook I believe we should place it under the `labelstudio/src/hooks` directory and name it `useImportFiles`

Reply:
assume you're referring to the `importFiles` function here. it is not meant to be a hook, just a utility that can be used in other hooks